### ISSUE TITLE
Get recipes from kubejs and mapping the emc

### ISSUE THREAD

**Issue by gjmhmm8** [TagnumElite/ProjectE-Integration#107]
Get recipes from kubejs and mapping the emc

**Comment by TagnumElite** [comment]
I don't believe this will be possible due to the open ended api that KubeJS suppplies.

**Comment by gjmhmm8** [comment]
kubejs can export all recipes easily and it can be useful in modpack

ServerEvents.recipes(event => {
  let recipes=[]
  event.forEachRecipe({},r=>{
		// console.log("recipe",r.json.toString())
    recipes.push(JSON.parse(r.json.toString()))
	})
  let recipe=JSON.stringify(recipes)
  console.log(recipe)
})

**Comment by xinxinsuried** [comment]
When I was creating a feature, I found that this method had an issue of incomplete export, but after replacing recipes with afterRecipes, there was no for each recipe method
